components/Dialogs/examples/DialogsAlertCustomizationViewController.swift

```diff
@@ -83,19 +88,72 @@ class DialogsAlertCustomizationViewController: MDCCollectionViewController {
     switch indexPath.row {
     case 0:
       didTapCenteredTitle()
+    case 1:
+      didTapCenteredTitleWithIcon()
+    case 2:
+      didTapNaturalTitleWithIcon()
+    case 3:
+      didTapRightTitleWithResizedIcon()
+    case 4:
+      didTapNoTitleWithIcon()
     default:
       print("No row is selected")
     }
   }
 
+  func sampleIcon(isStandardSize: Bool = true) -> UIImage? {
+    let bundle = Bundle(for: DialogsAlertCustomizationViewController.self)
+    return UIImage(
+      named: isStandardSize ? "outline_lock_black_24pt" : "baseline_alarm_on_black_48pt",
+      in: bundle, compatibleWith: nil)
+  }
+
   func didTapCenteredTitle() {
-    let alert = createMDCAlertController(title: "Dialog Title")
+    let alert = createMDCAlertController(title: "Center Aligned Title")
+    alert.titleAlignment = .center // todo: theme with themer when available
+    MDCAlertControllerThemer.applyScheme(alertScheme, to: alert)
+    self.present(alert, animated: true, completion: nil)
+  }
+
+  func didTapCenteredTitleWithIcon() {
+    let alert = createMDCAlertController(title: "Center Aligned Title")
+    alert.titleIcon = sampleIcon()
     alert.titleAlignment = .center // todo: theme with themer when available
+
+    MDCAlertControllerThemer.applyScheme(alertScheme, to: alert)
+    self.present(alert, animated: true, completion: nil)
+  }
+
+  func didTapNaturalTitleWithIcon() {
+
+    let alert = createMDCAlertController(title: "Natuarally Aligned Title")
+    alert.titleIcon = sampleIcon()
+    alert.titleAlignment = .natural // todo: theme with themer when available
+
+    MDCAlertControllerThemer.applyScheme(alertScheme, to: alert)
+    self.present(alert, animated: true, completion: nil)
+  }
+
+  func didTapRightTitleWithResizedIcon() {
+
+    let alert = createMDCAlertController(title: "Right Aligned Title")
+    alert.titleIcon = sampleIcon(isStandardSize: false)
+    alert.titleAlignment = .right // todo: theme with themer when available
+
+    MDCAlertControllerThemer.applyScheme(alertScheme, to: alert)
+    self.present(alert, animated: true, completion: nil)
+  }
+
+  func didTapNoTitleWithIcon() {
+
+    let alert = createMDCAlertController(title: nil)
+    alert.titleIcon = sampleIcon()
+
     MDCAlertControllerThemer.applyScheme(alertScheme, to: alert)
     self.present(alert, animated: true, completion: nil)
   }
 
-  private func createMDCAlertController(title: String) -> MDCAlertController {
+  private func createMDCAlertController(title: String?) -> MDCAlertController {
```

Comment:
_Tip 1_: Change the function signature to 
`createMDCAlertController(title: String?, iconStandardSize: Bool = true)`

Then in the `createMDCAlertController` function call `sampleIcon()` with respective value.

_Tip 2_: This can be marked `class` if you think it may be subclassed or `static` it doesn't rely on `self` at all.

Reply:
Not all examples are going to have an icon, and there are 2 kinds of icons, which will make the interface unnecessarily complicated.